exporter/coralogixexporter/config.go

```diff
@@ -27,9 +29,10 @@ const (
 
 // Config defines by Coralogix.
 type Config struct {
-	config.ExporterSettings      `mapstructure:",squash"`
-	exporterhelper.QueueSettings `mapstructure:"sending_queue"`
-	exporterhelper.RetrySettings `mapstructure:"retry_on_failure"`
+	config.ExporterSettings       `mapstructure:",squash"`
+	exporterhelper.QueueSettings  `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings  `mapstructure:"retry_on_failure"`
+	configgrpc.GRPCClientSettings `mapstructure:",squash"`
 
 	// The Coralogix logs ingress endpoint
 	Endpoint string `mapstructure:"endpoint"`
```

Comment:
The gRPC client settings object already has an Endpoint, use that.

Reply:
I want to stay with Endpoint configuration name to make the config file easier to use.
After the users put the endpoint we modify the GRPCClientSettings.
Can you explain why to remove "endpoint" ?

Reply:
There's no difference for your users, as you are squashing the configuration.

Reply:
Thanks, I will use it